docs/ChromeOS.md

```diff
@@ -19,4 +19,24 @@ Any shared folders will be avaliable under `/mnt/chromeos/`.
 
 Alternatively you could install a browser within the virtual machine itself.
 
+# Flashing the firmware (DFU mode) and USB access 
+
+By default Chrome OS prevents Linux from having access to most USB devices.
+This means an extra step is required to flash the firmware, once this is done you can follow the Linux instructions for flashing firmware.
+
+On your Chromebook in chrome visit `chrome://flags/#crostini-usb-allow-unsupported` and enable the flag. These flags are experimental features that can be turned on and off and some like this one affect what the Linux Beta can do. Enabling this flag allows you to give Linux access to any USB device.
+
+You will need to restart Linux for this to take effect. The easiest way to do this is to restart the Chromebook. 
+Shutdown from terminal does not seem to close gracefully and you lose your terminal history.
+
+Once Linux is back up and running update is as it should pull down some packages thanks to the flag that has just been enabled. 
```

Comment:
Should this be something along the lines of:

Once Linux is back up and running you should update update it. This should pull down some packages thanks to the flag that has just been enabled.

Reply:
Oh crikey looks like there was a difference between my brain and fingers. I have amended it now.

Reply:
We've all been there 😆